config/crd/bases/medusa.k8ssandra.io_cassandrabackups.yaml

```diff
@@ -8222,8 +8222,8 @@ spec:
                         properties:
                           additionalVolumes:
                             items:
-                              description: StorageConfig defines additional storage
-                                configurations
+                              description: AdditionalVolumes StorageConfig defines
```

Comment:
description seems a bit off when reading it.

Reply:
Yes, but this is generated automatically, there's not much I can do about this I think.